docs/how-to-register-processor.ftd

```diff
@@ -0,0 +1,34 @@
+-- ft.page: How to register your processor in `settings.py`
+
+Once you have defined your processors inside `processors.py`,
+you must register these processors before using them.
+
+To do that head over to your settings.py file and
+update `TEMPLATES` to incorporate your custom defined processors.
+
+-- ft.code: Update TEMPLATES inside your settings.py
+lang: py
+
+TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "DIRS": [],
+        "APP_DIRS": True,
+        "OPTIONS": {
+            "context_processors": [
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
+                "django.contrib.auth.context_processors.auth",
+                "django.contrib.messages.context_processors.messages",
+            ],
+        },
+    },
```

Comment:
Delete this bit.

Reply:
Okay

Reply:
In general when you get a suggestion/comment during review, instead of acknowledging like this, just incorporate it, and mark the comment as resolved. This okay was noise, not signal. 

In general use thumbs up emoji to say okay in other cases.

Comments leads to notifications, notifications should only be used for real things.